docs/bytecode.md

```diff
@@ -31,48 +31,48 @@ in the following code comments:
 The following attributes / types are subclasses of builtin machinery and call
 into the bytecode implementations in the Builtin Dialect.
 
-- `StableHLO_ArrayOfLayoutAttr`
-- `StableHLO_BoolElementsAttr`
-- `StableHLO_FlatSymbolRefArrayAttr`
-- `StableHLO_LayoutAttr`
-- `HLO_ComplexTensor`
-- `HLO_Complex`
-- `HLO_DimensionTensor`
-- `HLO_DimensionValue`
-- `HLO_Float32Or64`
-- `HLO_Float`
-- `HLO_Fp32Or64Tensor`
-- `HLO_FpOrComplexTensor`
-- `HLO_FpTensor`
-- `HLO_IntFpOrComplexTensor`
-- `HLO_IntOrFpTensor`
-- `HLO_IntTensor`
-- `HLO_Int`
-- `HLO_PredIntOrFpTensor`
-- `HLO_PredOrIntTensor`
-- `HLO_PredTensor`
-- `HLO_Pred`
-- `HLO_QuantizedIntTensor`
-- `HLO_QuantizedInt`
-- `HLO_QuantizedSignedInt`
-- `HLO_QuantizedUnsignedInt`
-- `HLO_SInt`
-- `HLO_ScalarIntTensor`
-- `HLO_StaticShapeTensor`
-- `HLO_TensorOrTokenOrTuple`
-- `HLO_TensorOrToken`
-- `HLO_Tensor`
-- `HLO_Tuple`
-- `HLO_UInt`
+  - `StableHLO_ArrayOfLayoutAttr`
+  - `StableHLO_BoolElementsAttr`
+  - `StableHLO_FlatSymbolRefArrayAttr`
+  - `StableHLO_LayoutAttr`
+  - `HLO_ComplexTensor`
+  - `HLO_Complex`
+  - `HLO_DimensionTensor`
+  - `HLO_DimensionValue`
+  - `HLO_Float32Or64`
+  - `HLO_Float`
+  - `HLO_Fp32Or64Tensor`
+  - `HLO_FpOrComplexTensor`
+  - `HLO_FpTensor`
+  - `HLO_IntFpOrComplexTensor`
+  - `HLO_IntOrFpTensor`
+  - `HLO_IntTensor`
+  - `HLO_Int`
+  - `HLO_PredIntOrFpTensor`
+  - `HLO_PredOrIntTensor`
+  - `HLO_PredTensor`
+  - `HLO_Pred`
+  - `HLO_QuantizedIntTensor`
+  - `HLO_QuantizedInt`
+  - `HLO_QuantizedSignedInt`
+  - `HLO_QuantizedUnsignedInt`
+  - `HLO_SInt`
+  - `HLO_ScalarIntTensor`
+  - `HLO_StaticShapeTensor`
+  - `HLO_TensorOrTokenOrTuple`
+  - `HLO_TensorOrToken`
+  - `HLO_Tensor`
+  - `HLO_Tuple`
+  - `HLO_UInt`
 
 **Special Cases:**
 
-- `StableHLO_ConvolutionAttributes`
-  - Despite its name,  is not an attribute and is not encoded.
+  - `StableHLO_ConvolutionAttributes`
+    - Despite its name,  is not an attribute and is not encoded.
     Rather, it is a dag which gets expanded into several attributes
     which are all encoded separately.
```

Comment:
If I recall correctly, indenting multiline bullets is not something that is currently done automatically by the tool, right? How should we approach this - can this only be fixed manually (doesn't have to be done in this PR, just trying to understand this better)? Also, is this something that the linter will flag in the future?

Reply:
Yeah. Well, I didn't realize that this indentation style was inconsistent across docs until now. It appears that spec.md was the only file that used 2-space indentation for all lists. The rest of these had none and now changing them to two creates this misalignment. 

So, there's some manual effort required for this no matter what (to make multiline list items left aligned). As such, the decision about whether to use `start_indented: true` should be either based on reducing the total amount of manual effort required (which is probably to use it---keep it `true`---because spec.md is a much larger document than the rest of these combined) or based on logic (which I believe leads to not using it because the extra indentation serves no real purpose and is actually harmful because it subtracts 2 characters from the available line limit, and arguably slows the writing process).

That is, I think using `start_indented: false` (no indentation for the list) is better if we were starting from scratch because it saves space and the only reason to keep it set `true` is because spec.md already uses 2-space indents.

You and your team are ultimately the authors here, so I think the decision is yours. Although the evidence suggests there's no strong preference to indent because not all documents do it, I'll honor your wishes if you want to do it that way for all these pages as well.

Reply:
This actually isn't that much work to align all these, either way, and I'll do it. So just confirm for me which style you prefer.